Yafc.Model/Serialization/ValueSerializers.cs

```diff
@@ -256,8 +256,10 @@ internal class TypeSerializer : ValueSerializer<Type> {
                 return null;
             }
 
-            s = reader.GetString();
-            Type? type = s is null ? null : Type.GetType(s);
+            if (s.StartsWith("YAFC.")) {
+                s = "Yafc." + s[5..];
+            }
+            Type? type = Type.GetType(s);
```

Comment:
Should we introduce file versioning? So we know when a file is migrated to the new namespace. Adn at some point, in the future, we can drop the conversion support and issue a warning that an older version of Yafc is required to improve the old/previous format?

Reply:
I didn't think much about versioning, beyond that I wanted the current version to write files that 0.6.x could read, in case I needed to go back and test older versions.

When we do need to update the version, we could do something with the first three lines, like the clipboard, or we can probably read the yafcVersion property.